### PR TITLE
Connect to API

### PR DESCRIPTION
**Pull request by Dachrono**
# 🎉 🎈 In this *Milestone* 🙋‍♂️ 🚩 ‼ :

- Follow the best GitHub flow. ✅
- Install linters. ✅
- No linters error. ✅
- Install redux toolkit. ✅ 
- Create redux logic folders. ✅
- Use of API. ✅
- Use of axios with createAsyncThunk. ✅
- Show book from API. ✅
- Add and remove new functions. ✅

### CODE REVIEW

File: src/redux/books/booksSlice.js [hunk 1]
```diff
@@ -1,43 +1,84 @@
...
+    const response = await axios.get(apiUrl);
+    return response.data;
+  } catch (error) {
+    throw new Error('Error al obtener los datos de la API');
+  }
+});
+
+export const apiPost = createAsyncThunk('books/apiPost', async (newBook) => {
+  try {
+    const response = await axios.post(apiUrl, newBook);
```
Comment (rustam-tolipov) [inline src/redux/books/booksSlice.js:17]: **You have done a great job!**
One more step, to get approved! 

- [ ] Your 'GET' requiest is working perfectly. The issue is with post request. Each time if I try to add a new book, it shows me `403` error.  Please fix the issue and make sure it's posting as expected. Possible solution woule be with your api key. Try to update your api key and check if it's posting correctly.

Screenshot 2023-07-21 at 09 28 10

**Review by rustam-tolipov — CHANGES_REQUESTED** [review]
Hello @Dachrono,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
_Your project is almost complete!_
_No Linters errors_
_Well-implemented code_
_Great readme_

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Check the comments under the review._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/redux/books/booksSlice.js (hunk 1 above)
Comment (Dachrono) [reply src/redux/books/booksSlice.js:17]: Solved

**Review by zainsadaqat — APPROVED** [review]
# Pull Request Approved ✅ 🎊 🎉 
## Hello @Dachrono  👋 

### Things to appreciate
- [ ] Implemented the missing things requested by the previous code reviewer 👍 

### Your project is complete!  There is nothing else to say other than it's time to merge it.

# Congratulations!!! 🎉 🥳 👏 🙌 🎊 ☑️ 



Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**